src/hooks/custom-hooks/useIsP2PBlocked.ts

```diff
@@ -0,0 +1,52 @@
+import { useMemo } from 'react';
+import { getBlockedType } from '@/utils';
+import { useGetAccountStatus } from '@deriv-com/api-hooks';
+import { api } from '..';
+
+// TODO: add other blocked statuses when implementing high risk scenarios
+/**
+ * A custom hook that returns the P2P blocked status and the status type.
+ * @returns {object} { isP2PBlocked: boolean, status: string }
+ */
+const useIsP2PBlocked = () => {
+    const { data: accountStatus } = useGetAccountStatus();
+    const { data: activeAccountData } = api.account.useActiveAccount();
```

Comment:
i also checked the network tab, this does not call active account list multiple times